src/Esfa.Vacancy.Register.Api/Controllers/ApprenticeshipsSearchController.cs

```diff
@@ -9,11 +9,11 @@
 namespace Esfa.Vacancy.Register.Api.Controllers
 {
     [ExceptionHandling]
-    public class ApprenticeshipSearchController : ApiController
+    public class ApprenticeshipsSearchController : ApiController
```

Comment:
I would keep this ApprenticeshipSearchController

Reply:
I thought we all agreed on this in the review meeting, also the Vacancy was renamed to Vacancies, it is consistent with routes.